### ISSUE TITLE
[Screen Reader-Home Page]: Screen reader is not providing the complete information about the Section Region Landmark for the Home Page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2628]
**User Experience:**
Screen reader users will not be able to access the Home Page if its Landmark regions are not defined in descriptive ways as users will not get information about the page contents.

**Test Environment:**
OS: Windows 10 RS2
Build: 15063.2
Browser: Edge

**Pre-Requisite:** Turn on the Narrator
**Steps to reproduce:**
1. Navigate to url:- https://makecode.com/
2. Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3-Verify whether proper Landmark regions are defined for the home page or not.

**Actual Behavior:**
Screen reader is not providing the complete information about the Section Region Landmark for the Home Page as Screen reader reads them as only Section region landmark.

**Expected Behavior:**
Screen reader should provide the complete information about the Section Region Landmark for the Home Page as Screen reader reads them with name so that users get the complete information.

**Note:**NVDA is not supporting the Landmark region on this Application.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Suggested Fix:**
1- ARIA roles/attributes must be used wherever possible to support AT’s in a better way. 
Roles: 
1.1 Landmarks Roles: All the sections should be aligned with respective landmark role 
e.g. 
<div id="header" role="banner">A banner image and introductory title</div> 
<div id="sitelookup" role="search">....</div> 
1.2) Headings Roles : All the heading elements should be aligned with respective role or role and aria-level 
e.g. 
<div role="heading">Global News items</div> 
... a list of global news with editorial comment... 
<div role="heading" aria-level="7">Jonagold/div> 
<p>Jonagold is a cross between the Golden Delicious and Jonathan varieties...</p> 
1.3) Roles associated with Labels: sections/regions of the webpage should be aligned with label’s  
e.g. 
<div role="application" aria-labelledby="p123" aria-describedby="info"> 
 
**Please refer attachment for more details.**
[MAS1.3.1_Home Page.zip](https://github.com/Microsoft/pxt/files/1193774/MAS1.3.1_Home.Page.zip) #2631

**Comment by microsoftsam** [comment]
Fixed on makecode.com

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ and bug still exists as Narrator is still not announcing the name or any information about the Section region landmarks.Hence Reactivated the bug and added the Active tag.

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is fixed now. Hence closing the bug
